llvm/lib/Target/X86/Disassembler/X86Disassembler.cpp

```diff
@@ -1171,7 +1186,7 @@ static int getInstructionID(struct InternalInstruction *insn,
         attrMask |= ATTR_EVEXB;
       // nf bit is the MSB of aaa
       if (nfFromEVEX4of4(insn->vectorExtensionPrefix[3]) &&
-          insn->opcodeType == MAP4)
+          (insn->opcodeType == MAP4 || isNFnotMap4(insn)))
```

Comment:
Merge this into a function like `isNF(insn)`?

Reply:
Doable.

Reply:
+1 for this suggestion.